nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java

```diff
@@ -110,7 +111,7 @@ private static String pad3Places(final long val) {
      */
     public static String formatDataSize(final double dataSize) {
         // initialize the formatter
-        final NumberFormat format = NumberFormat.getNumberInstance();
+        final NumberFormat format = NumberFormat.getNumberInstance(Locale.US);
```

Comment:
I'm not sure we want to force US locale as it's used in many places. Any reason for this change?

Reply:
yes it's because it seems like Google BigQuery requries dates in US format (how funny) ... Well, at least, that's why I remember having changed that globally (which is obviously not a good thing). Maybe i should use a different number format for the BigQuery part ...

Reply:
No, it's not the reason, I can't remember why. I will rollback that part.